simplq/src/components/pages/Home/LandingPage.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { useAuth0 } from '@auth0/auth0-react';
 import Header from 'components/common/Header';
-import CreateJoinForm from 'components/common/CreateJoinForm';
-import styles from './home.module.scss';
+import QueueForm from 'components/common/CreateJoinForm';
+import styles from './Home.module.scss';
```

Comment:
should we have `Page` in the css file also? `HomePage.module.scss`

Reply:
Every component should own its css.  That way it can be reused elsewhere. Right now `Home.module.scss` contains css for more than just `HomePage.jsx`. Once all separated, we might get rid of it completely.